src/platforms/kotlin-multiplatform/initialization-strategies.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: Initialization Strategies
+sidebar_order: 0
+description: "Different strategies how to initialize the Kotlin Multiplatform SDK."
+---
+
+When it comes to initializing a Kotlin Multiplatform SDK, there are three strategies to consider:
+ - [Shared initializer](#shared-initializer)
+ - [Platform specific initializers](#platform-specific-initializers)
+ - [Hybrid approach](#hybrid-approach)
+
+## Shared initializer
+
+The shared initializer approach involves initializing the SDK in your shared codebase, using the same configuration options for all platforms. This approach is ideal for projects that prioritize consistency across platforms and do not require platform-specific customizations.
+
+Using a shared initializer provides a single source of truth for your SDK's configuration options. This can simplify maintenance and debugging efforts, as you only need to update one codebase for all platforms.
+
+Inside a Kotlin file in your commonMain e.g. `AppSetup.kt` or however you want to call it and create a function that will initialize the SDK.
+Afterwards you can call it in an early lifecycle stage in your platforms.
```

Comment:
```suggestion
The shared initializer approach involves initializing the SDK in your shared codebase, using the same configuration options for all platforms. This approach is ideal for projects that prioritize consistency across platforms and don't require platform-specific customizations.

Using a shared initializer provides a single source of truth for your SDK's configuration options. This can simplify maintenance and debugging efforts, as you only need to update one codebase for all platforms.

To initialize the SDK, create a Kotlin file in your commonMain e.g. `AppSetup.kt`, (or whatever you decide to call it), and create an initialization function. You'll then be able to call it in an early lifecycle stage in your platforms.
```

Not sure I understood this correctly, but this section read like there was missing information originally. (What do we want the user to do inside their Kotlin file?)

Reply:
The action that the user needs to do in their Kotlin file (AppSetup.kt) refers to the code bit that is shown immediately after this text.